test/WebSites/OutOfProcessWebSite/Startup.cs

```diff
@@ -84,5 +86,25 @@ public Task RestrictedNTLM(HttpContext context)
 
         public Task UpgradeFeatureDetection(HttpContext context) =>
             context.Response.WriteAsync(context.Features.Get<IHttpUpgradeFeature>() != null? "Enabled": "Disabled");
+
+        public Task CheckRequestHandlerVersion(HttpContext context)
+        {
+            // We need to check if the aspnetcorev2_outofprocess dll is loaded by iisexpress.exe
+            // As they aren't in the same process, we will try to delete the file and expect a file
```

Comment:
You can use Process.Modules on the test side to see what modules are loaded into iiexpress.

Reply:
I don't see a good way to get the iisexpress process associated with the test. The Process is private: https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs#L31